scss/components/_control-bar.scss

```diff
@@ -160,8 +160,8 @@
       cursor: default;
     }
 
-    .oo-cast {
-        width: 45px;
+    .oo-cast, .oo-cast-air {
```

Comment:
I think we can use the same class for Chromecast and Airplay and just make a difference on them using the Id attribute on each element.

Reply:
We don't use ids for the ControlBar buttons, so it would not be consistent to add id to Airplay and Chromecast buttons. I 'd prefer not to use id also because of unit tests fixing after that =)